source/core_json.c

```diff
@@ -1297,7 +1297,7 @@ static bool_ nextKeyValuePair( const char * buf,
  * @param[in] max  size of the buffer.
  * @param[in] query  The object keys and array indexes to search for.
  * @param[in] queryLength  Length of the key.
- * @param[out] outValue  A pointer to receive the address of the value found.
+ * @param[out] outValue  A pointer to receive the index of the value found.
```

Comment:
Maybe call it `outOffset` or `outValueOffset`?

Reply:
There are already something of a convention that a size_t value or outValue is an index.  I don't want to make sweeping changes for only negligible benefit.